angular/projects/public-nrpti/src/app/records/records-list/records-list.component.ts

```diff
@@ -224,7 +336,23 @@ export class RecordsListComponent implements OnInit, OnDestroy {
     this.tableTemplateUtils.navigateUsingParams(this.tableData, ['records']);
   }
 
-  checkboxChange() {}
+  /**
+   * Open advanced filters panel.
+   *
+   * @memberof RecordsListComponent
+   */
+  openAdvancedFilters(): void {
+    this.showAdvancedFilters = true;
+  }
+
+  /**
+   * Close advanced filters panel.
+   *
+   * @memberof RecordsListComponent
+   */
+  closeAdvancedFilters(): void {
+    this.showAdvancedFilters = false;
+  }
```

Comment:
Why not just change the variable in the HTML.

Reply:
at some point in past I read an article or post, from angular I think, about preferring logic in the TS over HTML when possible.

But I also just find I prefer this style, and it seems cleaner/easier.